clients/python-wrapper/lakefs/config.py

```diff
@@ -0,0 +1,72 @@
+import os
+import yaml
+from typing import NamedTuple
+
+from lakefs_sdk import (
+    Configuration,
+)
+from lakefs.exceptions import NoAuthenticationFound
+
+_LAKECTL_YAML_PATH = os.path.expanduser(os.path.join("~", ".lakectl.yaml"))
```

Comment:
```
from pathlib import Path
```

```suggestion
_LAKECTL_YAML_PATH =Path.home() / '.lakectl.yaml'
```

Reply:
Your version will not work on windows (fixed)